JiraReporter/RandomGenerator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace JiraReporter
+{
+    class RandomGenerator
+    {
+        public static string RandomString(int length, Random random)
+        {
+            var chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+            var randomString = new string(
+                Enumerable.Repeat(chars, length)
+                .Select(s => s[random.Next(s.Length)])
+                .ToArray());
+
+            return randomString;
```

Comment:
Use this instead:
`return Path.GetRandomFileName().Replace(".", string.Empty);`